DS/DS2/MainStackImplementation.java

```diff
@@ -0,0 +1,77 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+//class used to define a stack and its operation
+class Node {
+	Node addressOfNextNode,top;
+	Object listInformation;
+}
+class StackUsingLinkedList extends Node{
+	
```

Comment:
Please check following link for inheritance 
http://www.tutorialspoint.com/java/java_inheritance.htm
http://examples.javacodegeeks.com/java-basics/java-inheritance-example/